addon/mixins/authenticated-route-mixin.js

```diff
@@ -33,12 +33,25 @@ export default Mixin.create({
   */
   session: service('session'),
 
+  _cookies: computed(function() {
+    return getOwner(this).lookup('service:cookies');
+  }),
```

Comment:
We should lookup the service inside of the sessionAuthenticated method so that we don't add any unnecessary properties via the mixin.